htmd/adaptive/adaptiverun.py

```diff
@@ -276,29 +276,41 @@ def __setattr__(self, key, value):
         uisetattr(self, key, value, all)
 
     def _algorithm(self):
+
         logger.info('Postprocessing new data')
-        datalist = simlist(glob(path.join(self.datapath, '*', '')), glob(path.join(self.inputpath, '*', 'structure.pdb')),
-                           glob(path.join(self.inputpath, '*', '')))
+
+        if self.app.get_class() == 'PmemdLocal':
+            datalist = simlist(glob(path.join(self.datapath, '*', '')),
+                               glob(path.join(self.inputpath, '*', 'structure.prmtop')),
+                               glob(path.join(self.inputpath, '*', '')))
+        else:
+            datalist = simlist(glob(path.join(self.datapath, '*', '')),
+                               glob(path.join(self.inputpath, '*', 'structure.pdb')),
+                               glob(path.join(self.inputpath, '*', '')))
         filtlist = simfilter(datalist, self.filteredpath, filtersel=self.filtersel)
 
+        # calculate metrics for tica projection
         if hasattr(self, 'metricsel2') and self.metricsel2 is not None:
             proj = MetricDistance(self.metricsel1, self.metricsel2, metric=self.metrictype)
         else:
             proj = MetricSelfDistance(self.metricsel1, metric=self.metrictype)
+
         metr = Metric(filtlist, skip=self.skip)
         metr.projection(proj)
         data = metr.project()
 
         #if self.contactsym is not None:
         #    contactSymmetry(data, self.contactsym)
 
+        # projecting data with tICA
         data.dropTraj()
         if self.ticadim > 0:
             tica = TICA(data, int(max(2, np.ceil(20/self.skip))))
             datadr = tica.project(self.ticadim)
         else:
             datadr = data
 
+        # what is this K definition???
         K = int(max(np.round(0.6 * np.log10(datadr.numFrames/1000)*1000+50), 100))  # heuristic
```

Comment:
It's a heuristic for determining how many clusters to use for the Markov model. See the SI Fig 1. of my paper
http://pubs.acs.org/doi/suppl/10.1021/acs.jctc.6b00049